android.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -eu
+#
+# Replace jdk.table xml file
+# For Android Studio to detect the correct JDK path
+# we need to preset the path in the jdk.table.xml
+#
+
+OPTIONS_DIR="$HOME/.AndroidStudio2.2/config/options"
+
+if [[ ! -f ${OPTIONS_DIR}/jdk.table.xml ]]; then
+  mkdir -p ${OPTIONS_DIR}
+  cp /app/AndroidStudioConfig/jdk.table.xml ${OPTIONS_DIR}/jdk.table.xml
+fi
+
+exec /app/bin/studio.sh $@
```

Comment:
@krnowak: Yeah, expansions whih include `@` are special-cased, and always expand “correctly” in the sense that POSIX defines that `"$@"` is to be expanded as `"$1" "$2" ... "$N"`. This treatment is also applied when expanding array variables (in shells that support them), e.g. `handle_data "${array_variable[@]}"`, which makes arrays super convenient for collecting command line parameters :wink: